RoddyCore/test/de/dkfz/roddy/config/ConfigurationValueTest.groovy

```diff
@@ -13,15 +13,26 @@ import org.junit.Test
  * Created by heinold on 14.07.16.
  */
 @CompileStatic
-class ConfigurationValueTest extends GroovyTestCase {
-
-    // Test for determineTypeOfValue is in RoddyConversionHelperMethodsTest.groovy
-    // The method is directly bound to several methods of the conversion class and has nearly no
-    // logic of its own. So we put it there.
+class ConfigurationValueTest {
```

Comment:
This configuration variable evaluation is extremely important (what happens if the evaluation is wrong and say 1000 WGS samples are processed the wrong way?!

I suggest we write many more tests -- and, actually, for the old version! After that we test that the behaviour of preserved for this new version. I also suggest we read in a CO configuration tree and few plugin configurations, and check before and after the change, to ensure that things go as they should. I feel extreme uncomfortable changing this code without tests.